reindexer/reindex_worker/src/main/scala/uk/ac/wellcome/platform/reindex_worker/services/ReindexService.scala

```diff
@@ -1,37 +1,32 @@
 package uk.ac.wellcome.platform.reindex_worker.services
 
-import javax.inject.Inject
 import com.amazonaws.services.dynamodbv2.AmazonDynamoDB
 import com.gu.scanamo.error.DynamoReadError
 import com.gu.scanamo.query._
 import com.gu.scanamo.syntax._
 import com.gu.scanamo.{Scanamo, _}
 import com.twitter.inject.Logging
-import com.twitter.inject.annotations.Flag
+import javax.inject.Inject
```

Comment:
I've used Guice `@Inject` elsewhere as i've made the assumption we're using it for DI everywhere. Should We be using `javax.inject.Inject` instead for better compatibility with other DI libraries?

Reply:
I don't think javax.inject is introduced in this change, but the import moved.  Guice should be [JSR330 compatible](https://github.com/google/guice/wiki/JSR330), so I'd prefer it except if we are using Guice inject everywhere else, or using it via Scala introduces doubts over compatibility.

Looks like `com.google.inject.Inject` wins 37 to 2 for `javax.inject.Inject` (here and SnapshotService).  Perhaps another PR to make it Guice Inject everywhere?